lib/server/server.js

```diff
@@ -23,15 +24,12 @@ function execute(port) {
   const glob = require('glob');
   const chalk = require('chalk');
   const translate = require('./translate.js');
-  const versionFallback = require('./versionFallback');
```

Comment:
I am assuming this is removed because it is unused.

Reply:
yes this is unused.